tests/test_bundle.py

```diff
@@ -18,11 +19,16 @@
     wait_fixed,
 )
 
+
 log = logging.getLogger(__name__)
 
 KSVC = create_namespaced_resource(
     group="serving.knative.dev", version="v1", kind="Service", plural="services"
 )
+KNATIVE_SERVING_SERVICE = "services.serving.knative.dev"
+KNATIVE_OPERATOR_METADATA = yaml.safe_load(Path("./charms/knative-operator/metadata.yaml").read_text())
+KNATIVE_OPERATOR_IMAGE = KNATIVE_OPERATOR_METADATA["resources"]["knative-operator-image"]["upstream-source"]
+KNATIVE_OPERATOR_RESOURCES = {"knative-operator-image": KNATIVE_OPERATOR_IMAGE}
```

Comment:
I believe you also need the webhook image here

Reply:
You would be correct :)  Added